CVLByExample/summary/Library/DirectSummary/certora/specs/AllDirect.spec

```diff
@@ -0,0 +1,41 @@
+/**
+ * Every function being called (from solidity) is given its own direct summarization.
+ */
```

Comment:
For each function called from Test.sol a summary is provided

Reply:
done